src/encoded/tests/test_search_views.py

```diff
@@ -1,9 +1,9 @@
 import pytest
 
-from encoded.tests.features.conftest import app_settings, app, workbook
+from encoded.tests.features.conftest import app_settings, app, index_workbook
 
 
-def test_search_views_search_view_with_filters(workbook, testapp):
+def test_search_views_search_view_with_filters(index_workbook, testapp):
```

Comment:
> -m “indexing” - all tests now pytest.mark.indexing  - includes test_indexing, test_batchdownload, and test_search_views.

Were these supposed to be marked as indexing? Looks like they are still running in `non bdd non indexing tests`.

Reply:
it looks like maybe the search_views tests didn't get pytest.marked as indexing?

Reply:
fixed